canvas_data_warehouse/services.py

```diff
@@ -25,45 +25,51 @@ class CanvasDataService:
     BY_TOTAL_WORKLOAD = 3
     BY_PER_CAPITA_WORKLOAD = 4
 
-    staff_grouping_type = [
-        {'type': BY_DEPARTMENT, 'name': '按学院'},
-        {'type': BY_STAFF_TYPE, 'name': '按人员类别'},
-        {'type': BY_STAFF_TITLE, 'name': '按职称'},
-        {'type': BY_HIGHEST_DEGREE, 'name': '按最高学位'},
-        {'type': BY_AGE_DISTRIBUTION, 'name': '按年龄分布'}
-    ]
-    trainee_grouping_type = [
-        {'type': BY_DEPARTMENT, 'name': '按学院'},
-        {'type': BY_STAFF_TITLE, 'name': '按职称'},
-        {'type': BY_AGE_DISTRIBUTION, 'name': '按年龄分布'}
-    ]
-    training_hours_grouping_type = [
-        {'type': BY_TOTAL_STAFF_NUM, 'name': '按总人数'},
-        {'type': BY_TOTAL_TRAINING_HOURS, 'name': '按总培训学时'},
-        {'type': BY_PER_CAPITA_TRAINING_HOURS, 'name': '按人均培训学时'},
-        {'type': BY_TOTAL_WORKLOAD, 'name': '按总工作量'},
-        {'type': BY_PER_CAPITA_WORKLOAD, 'name': '按人均工作量'}
-    ]
+    staff_grouping_type = (
```

Comment:
Constant name should be UPPER_CASE, and the name should follow the rule of `XXX_CHOICES`